src/sentry/static/sentry/app/utils/sdk.js

```diff
@@ -0,0 +1,84 @@
+let _addBreadcrumb = function() {};
+let _captureException = function() {};
+let _captureMessage = function() {};
+let _showReportDialog = function() {};
+let _lastEventId = function() {};
+
+document.addEventListener('ravenLoaded', function() {
+  if (window.Raven) {
```

Comment:
Doesn't `ravenLoaded` mean that `window.Raven` is always available? If not, we should probably do something like:

```js
var retries = 5;

function tryAttachRaven () {
  if (window.Raven) {
    // code below
  } else if (retries > 0) {
    retries -= 1;
    setTimeout(tryAttachRaven, 100);
  }
}

document.addEventListener('ravenLoaded', tryAttachRaven)
```